string_unique/README.md

```diff
@@ -42,4 +42,13 @@ Run o(n^2) by iterating across the same string multiple times, starting 1 charac
 Autobuilding mvn - seems slow the first time but...
 ```
 mvn archetype:generate -DgroupId=net.mtnboy -Dpackage=net.mtnboy.StringUnique -DartifactId=string-app -DarchetypeArtifactId=maven-archetype-quickstart -DinteractiveMode=false
-```
\ No newline at end of file
+```
+
+### Time to complete
+Approximately 2 hours, with two minor interruptions. Lots of mvn learning, had to restructure code for mvn after this. Spent way too long thinking about time vs. memory complexity (favored memory heavily). 
+
+### Things I learned
+* chars vs. Characters - seems like I can't instantiate Sets of basic data types - must be classes?
+* mvn setup, including the default example on apache is not valid. New mvn requires package.
+* Basic testing in java
+* That an editor that fixes imports for me is useful. Emacs not really cutting it. But making me think a bit.
```

Comment:
Finally! Get out of the stone age.

Reply:
reedited this in intellij. Not thrilled with the refactor suggestions and the annotations inline. Dynamically reformatting code made it harder to read. Maybe turn off a bit of that?